metadata-ingestion/src/datahub/ingestion/source/sql/sql_common.py

```diff
@@ -317,6 +326,7 @@ def __init__(self, config: SQLAlchemyConfig, ctx: PipelineContext, platform: str
         self.config = config
         self.platform = platform
         self.report: SQLSourceReport = SQLSourceReport()
+        self.profile_metadata_info: ProfileMetadataInfo = ProfileMetadataInfo()
```

Comment:
What do you think about more concisely calling this `ProfileMetadata` or `ProfileInfo`? I don't really understand what metadata + info is conveying